web_warning_on_save/static/src/js/web_warning_on_save.js

```diff
@@ -0,0 +1,23 @@
+openerp.web_warning_on_save = function (instance) {
+
+	instance.web.FormView.include({
+		on_button_save: function(e) {
+//			inherit method to be able to display an alert if
+//			method called returns a string
+			var id = 0;
+			var self = this;
+			self._super.apply(this, arguments).done(function(id){
```

Comment:
return that, otherwise you lose the deferred object for other functions inheriting `on_button_save`